docs/en/sql-reference/aggregate-functions/reference/timeSeriesLastToGrid.md

```diff
@@ -0,0 +1,69 @@
+---
+description: 'Aggregate function that calculates PromQL-like last over time series data on the specified grid.'
+sidebar_position: 225
+slug: /sql-reference/aggregate-functions/reference/timeSeriesLastToGrid
+title: 'timeSeriesLastToGrid'
+---
+
+Aggregate function that takes time series data as pairs of timestamps and values and calculates [PromQL-like last_over_time](https://prometheus.io/docs/prometheus/latest/querying/functions/#aggregation_over_time) from this data on a regular time grid described by start timestamp, end timestamp and step. For each point on the grid the samples for calculating `last_over_time` are considered within the specified time window.
+
+Parameters:
+- `start timestamp` - specifies start of the grid
+- `end timestamp` - specifies end of the grid
+- `grid step` - specifies step of the grid in seconds
+- `staleness` - specified the maximum "staleness" in seconds of the considered samples
+
+Arguments:
+- `timestamp` - timestamp of the sample
+- `value` - value of the time series corresponding to the `timestamp`
+
+Return value:
+`last_over_time` values on the specified grid as an `Array(Nullable(Float64))`. The returned array contains one value for each time grid point. The value is NULL if there are not enough samples within the window to calculate the `last_over_time` value for a particular grid point.
```

Comment:
Looks like https://clickhouse.com/docs/sql-reference/aggregate-functions/reference/timeSeriesResampleToGridWithStaleness is already doing this but isn't named properly
Let's add an alias